bluebottle/suggestions/views.py

```diff
@@ -1,14 +1,14 @@
 from datetime import date
 from rest_framework import generics, status, response
-from rest_framework.permissions import IsAuthenticated
+from rest_framework.permissions import AllowAny
 
 from bluebottle.suggestions.models import Suggestion
 from bluebottle.suggestions.serializers import SuggestionSerializer
 
 
 class SuggestionList(generics.ListCreateAPIView):
     queryset = Suggestion.objects.all()
-    permission_classes = (IsAuthenticated,)
+    permission_classes = (AllowAny,)
     serializer_class = SuggestionSerializer
```

Comment:
In still working on this, its not meant to be merged yet. Ill add Do Not Merge to the title

Sent from my iPhone

> On 14 jun. 2016, at 13:24, eodolphi <notifications@github.com> wrote:
> 
> In bluebottle/suggestions/views.py:
> 
> >  
> >  from bluebottle.suggestions.models import Suggestion
> >  from bluebottle.suggestions.serializers import SuggestionSerializer
> >  
> >  
> >  class SuggestionList(generics.ListCreateAPIView):
> >      queryset = Suggestion.objects.all()
> > -    permission_classes = (IsAuthenticated,)
> > +    permission_classes = (AllowAny,)
> >      serializer_class = SuggestionSerializer
> Same here, not anybody should be able to read the list of suggestions. Anybody should be able to create them though.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>